Priyaa/HTML Basics Remake/2.Table Remake/POKEMON.html

```diff
@@ -0,0 +1,53 @@
+<!DOCTYPE html>
+<html>
+	<head>
+		<title>Assignment - Table Remake</title>
+	</head>
+	<body>
+		<h1>Pokemon Chart</h1>
+		<table border="1">
```

Comment:
Switch to HTML5. Refer to this: [HTML \<table> border Attribute](https://www.w3schools.com/tags/att_table_border.asp)